source/404.html.md.erb

```diff
@@ -0,0 +1,10 @@
+---
+title: "404"
+last_reviewed_on: 2023-12-19
+review_in: 6 months
+hide_in_navigation: true
+---
+
+# 404 - file not found
+
+## Please use the navigation or reach out for help on [slack](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC)
```

Comment:
```suggestion
## If you typed the web address, check it is correct.

If you pasted the web address, check you copied the entire address.

You can contact us on our [GOV.UK One Login Slack channel](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC) if you need help.
```

To match the [GOV.UK Design System guidance](https://design-system.service.gov.uk/patterns/page-not-found-pages/) :)